### DISCUSSION TITLE
Is there a way to see what arguments kobold uses internally?

### DISCUSSION THREAD

**Discussion by tzwel** [LostRuins/koboldcpp#1079] [Q&A] [answered]
I want to get a command with arguments already set. Basically, _click out everything in the gui_ > _get a copy-pastable generated command_ so I don't have to type everything out

I see that it outputs `Namespace(benchmark=None, blasbatchsize=512, blasthreads=5, chatcompletionsadapter=None, config=None, contextsize=4096, debugmode=0...` but it would be tedious to correct manually

**Comment by LostRuins** [comment, marked as answer]
Unfortunately there is no way to directly covert a GUI setup into a command line argument. However, you can export the GUI settings as a kcpps config file, and then import it via command line with `--config`

**Reply by tzwel** [reply]
Thanks, this will do
